landing.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title> Landing Page</title>  
+    </head>
+    
+
+    <h1>Landing Page</h1>
+    <form>
+        <label for="username">username:</label><br>
```

Comment:
The value of the `for` attribute is incorrect. You should do further research on how this attribute works to figure out what the correct value is.